### PR TITLE
Genloc icc-api from icure backend master branch

### PR DESCRIPTION
**Pull request by phil1618**
ICure master commit : dbd6878fb17b27d376b17878e6b6f797049bdf3a

### CODE REVIEW

File: icc-api/LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-                                 Apache License
+ Apache License
```
Comment (cebamps) [inline icc-api/LICENSE:1]: What happened here? ;)

Comment (phil1618) [reply icc-api/LICENSE]: almost fixed now :-)

**Comment by phil1618** [comment]
@cebamps Thanks !

**Comment by phil1618** [comment]
@cebamps @gpiroux I'm compiling medirpirng prod with this branch to validate

**Comment by phil1618** [comment]
@cebamps @gpiroux It compiles just fine.

**Pull request merged**